newIDE/app/src/ObjectEditor/Editors/SpriteEditor/AnimationPreview/index.js

```diff
@@ -39,6 +40,7 @@ export default class AnimationPreview extends Component<Props, State> {
     currentFrameIndex: 0,
     currentFrameElapsedTime: 0,
     paused: false,
+    fps: parseFloat((1/this.props.timeBetweenFrames).toFixed(0)),
```

Comment:
Prefer math.Round to toFixed(0), it's more explicit that you're rounding the number